AlphaWallet/Tokens/Collectibles/Coordinators/NFTAssetSelectionCoordinator.swift

```diff
@@ -51,9 +51,9 @@ class TokenCardSelectionCoordinator: Coordinator {
     }
 }
 
-extension TokenCardSelectionCoordinator: TokenCardSelectionViewControllerDelegate {
+extension NFTAssetSelectionCoordinator: NFTAssetSelectionViewControllerDelegate {
```

Comment:
@oa-s @eviltofu terminology such as "token card" came from TokenScript (for context). But still it's good to rename those. We might bring back the "token card", "token view" etc names when we have better separation for TokenScript